public/style.css

```diff
@@ -78,10 +78,42 @@ img {
     display: grid;
     place-items: center;
 }
+.button {
+    background-color: darkslategray;
+    border: none;
+    color: white;
+    padding: 15px 32px;
+    text-align: center;
+    font-size: 20px;
+}
+.button:hover {
+    filter: drop-shadow(0 0 0.45rem darkslategrey);
+}
 /*this styles the admin page*/
 .adminPg {
     display: grid;
     place-items: center;
+    padding-left: 15%;
+    padding-right: 15%;
+}
+/*this gives a visual cue to the user that their input is valid or invalid*/
+input:invalid {
+    border: 2px solid red;
+    filter: drop-shadow(0 0.2rem 0.25rem red);
+}
+input:valid {
+    border: 2px solid green;
+}
+/*this styles the table in the admin page so it's easier to read*/
+table {
+    border-collapse: collapse;
+    width: 100%;
+}
+th, td {
+    padding: 8px;
+    text-align: left;
+    border-bottom: 1px solid darkslategray;
+}
+tr:hover {
+    background-color: white;
 }
-/*#error and #success will need styling for error/success messages*/
```

Comment:
my bads i aiint mean to take that out haley is adding in the invaild and vaild css